### PR TITLE
Warning to setup bridging-header for Swift development

### PR DESCRIPTION
**Pull request by brunobowden**
- Warns on setting up bridging header alongside warning about opening .xcworkspace

### CODE REVIEW

**Comment by brunobowden** [comment]
@advayDev1, @confile - PTAL

File: FAQ.md [hunk 1]
```diff
@@ -100,6 +100,24 @@ sourceSets {
...
+Within that bridging header, include the file needed for using the JRE and any classes that you'd like
+to access from Swift code.
+
+```
+// File: IOS-APP-bridging-header.h
+
+// Required for Swift initialization of Java objects (they all inherit from JavaObject)
+#import "JreEmulation.h"
+
+// Swift accessible j2objc translated classes, referenced from `build/j2objcSrcGen`
```
Comment (advayDev1) [inline FAQ.md]: no, they should be references solely from j2objcOutputs/src (not sure what the full path is).
j2objcSrcGen is our intermediate implementation detail

**Comment by advayDev1** [comment]
LGTM with comment

File: FAQ.md (hunk 1 above)
Comment (brunobowden) [inline FAQ.md]: Changed comment to `build/j2objcOutputs/src/main/objc` - The podspec is pointing to the wrong place, so I'll fix that in another commit.

**Pull request merged**